src/components/menubar/MenuBarTime.tsx

```diff
@@ -3,7 +3,7 @@ import { useState } from 'react';
 import { useInterval } from 'use-interval';
 
 const MenuBarTime = () => {
-  const [time, setTime] = useState(new Date());
+  const [time, setTime] = useState(() => new Date());
```

Comment:
This optimisation isn't really needed, as its pretty cheap for the browser to do